modules/ROOT/partials/integrations/svelte-quick-start.adoc

```diff
@@ -114,7 +151,7 @@ endif::[]
 ifeval::["{productSource}" == "zip"]
 
 . Open `+src/App.svelte+` and add:
-* An `+import+` statement for the {productname} component.
+* An `+import+` statement for the {productname} Svelte component.
```

Comment:
I would like to suggest we update the wording around this...
Currently if we follow the steps, the `App.svelte` does not work, as we needs to instruct the integrator to replace the contents of the `App.svelte` auto-generated content with the proposed editor config ect as you mention below.

Reply:
This is what im referring to.
`<script>
  import svelteLogo from './assets/svelte.svg'
  import viteLogo from '/vite.svg'
  import Counter from './lib/Counter.svelte'
</script>

<main>
  <div>
    <a href="https://vitejs.dev" target="_blank" rel="noreferrer">
      <img src={viteLogo} class="logo" alt="Vite Logo" />
    </a>
    <a href="https://svelte.dev" target="_blank" rel="noreferrer">
      <img src={svelteLogo} class="logo svelte" alt="Svelte Logo" />
    </a>
  </div>
  <h1>Vite + Svelte</h1>

  <div class="card">
    <Counter />
  </div>

  <p>
    Check out <a href="https://github.com/sveltejs/kit#readme" target="_blank" rel="noreferrer">SvelteKit</a>, the official Svelte app framework powered by Vite!
  </p>

  <p class="read-the-docs">
    Click on the Vite and Svelte logos to learn more
  </p>
</main>

<style>
  .logo {
    height: 6em;
    padding: 1.5em;
    will-change: filter;
    transition: filter 300ms;
  }
  .logo:hover {
    filter: drop-shadow(0 0 2em #646cffaa);
  }
  .logo.svelte:hover {
    filter: drop-shadow(0 0 2em #ff3e00aa);
  }
  .read-the-docs {
    color: #888;
  }
</style>`

Reply:
Good point! The instructions given were incorrect. I have changed the wording now to say "replace the contents".